arvo/clay/examples.md

```diff
@@ -0,0 +1,841 @@
+(pure:m !>(~))
+```
+
+Save to `ted/multi-change.hoon`, `|commit %home`, and run:
+
+```
+> -multi-change
++ /~zod/home/37/foo/txt
++ /~zod/home/37/bar/txt
++ /~zod/home/37/baz/txt
+- /~zod/home/38/foo/txt
+- /~zod/home/38/bar/txt
+- /~zod/home/38/baz/txt
+```
+
+# Manage Mounts
+
+See the [Manage Mounts](@/docs/arvo/clay/tasks.md#manage-mounts) section of the [API Reference](@/docs/arvo/clay/tasks.md) document for general details.
+
+## %boat
+
+Here we'll look at requesting the list of existing mount points on a ship by sending Clay a `%boat` `task` and receiving a `%hill` `gift`.
+
+Using the `send-task-take-gift.hoon` thread, let's make such a request:
+
+```
+> -send-task-take-gift [%boat ~]
+[%hill p=~[%home]]
+```
+
+## %mont
+
+Here we'll look at mounting `desk`s, directories and files to unix by sending Clay a `%mont` `task`.
+
+Using the `send-task.hoon` thread, let's first try mounting our `%kids` desk:
+
+```
+> -send-task [%mont %kids [our %kids da+now] /]
+```
+
+If you look in your pier, you should now see a `kids` folder which contains the contents of that `desk`.
+
+If we make a `%boat` request as detailed in the [%boat](#boat) section, we'll now see the mount point listed:
+
+```
+> -send-task-take-gift [%boat ~]
+[%hill p=~[%kids %home]]cruz gift. It looks like:
+
+[%cruz cez=(map @ta crew)]  ::  permission groups
+The cez is just a map from group name to crew
+```
+
+Note the mount point doesn't need to match a `desk`, file or directory. We can also do:
+
+```
+> -send-task [%mont %wibbly-wobbly [our %home da+now] /]
+```
+
+And you'll now see that there's a `wibbly-wobbly` folder with the contents of the `%home` `desk`. You'll also notice we can mount the same file or directory more than once. There's no problem having `%home` mounted to both `home` and `wibbly-wobbly`. The only requirement is that their mount points be unique.
+
+Let's try mounting a subdirectory and a single folder:
+
+```
+> -send-task [%mont %gen [our %home da+now] /gen]
+> -send-task [%mont %hi [our %home da+now] /gen/hood/hi]
+```
+
+If you look in your pier you'll now see a `gen` folder with the contents of `/gen` and a `hi.hoon` file by itself. Notice how the file extension has been automatically added.
+
+## %ogre
+
+Here we'll look at unmounting `desk`s, directories and files by sending Clay a `%ogre` `task`.
+
+Using the `send-task.hoon` thread, let's unmount what we mounted in the [%mont](#mont) section. First we'll unmount the `%kids` desk:
+
+```
+-send-task [%ogre %kids]
+```
+
+Our custom mount point `%wibbly-wobbly`:
+
+```
+-send-task [%ogre %wibbly-wobbly]
+```
+
+And the single `hi.hoon` we previously mounted by specifying its mount point `%hi`:
+
+```
+-send-task [%ogre %hi]
+```
+
+If we specify a non-existent mount point it will fail with an error printed to the dojo like:
+
+```
+> -send-task [%ogre %kids] 
+[%not-mounted %kids]
+```
+
+If we give it an unmounted `beam` it will not print an error but still won't work.
+
+## %dirk
```

Comment:
Maybe mention this corresponds to `|commit`

Reply:
have added note about `|commit`